### PR TITLE
Add Wreckfest

### PR DESCRIPTION
**Pull request by WGOS**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [x] Does your server use a custom docker image?
    * [x] Have you tried to use a generic image?
      **Generic image on ubnuntu-18.04 uses Wine 3.0 that doesn't work with Wreckfest, so Wine 4.0 on Debian needs to be used**  
     I also installing wine-mono and wine-gecko on startup (via entrypoint.sh)
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

**Comment by parkervcp** [comment]
You could PR a Wine 4.0 image to my repo.

I also haven't rebuilt the wine image recently and it may actually

**Comment by WGOS** [comment]
Ubuntu 18.04 contains only 3.0 in their default repo.
My wine-4.0 is WIP, I planned to add some features like steamcmd support and environment variables for installing libs like vcredist etc.

**Comment by parkervcp** [comment]
I want to hold on this until a generic image is available then. I prefer to have them all in one place, if that makes any sense. Plus others probably want to use it.

**Comment by WGOS** [comment]
Wine 4.0 is included by default in Ubuntu 19.10 release though

**Comment by parkervcp** [comment]
I think the next release of ubuntu I will base images off will be 20.04 as it will be LTS.

I am not opposed to having it be Debian Based.

**Comment by parkervcp** [comment]
I'll take a run at this today.

**Comment by WGOS** [comment]
Now it's with docker image from your repo

**Comment by parkervcp** [comment]
@WGOS A better installer using steamcmd - https://github.com/parkervcp/eggs/blob/master/scripts/steamcmd_installer.sh

File: wreckfest/egg-wreckfest.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"Server connected to Steam\",\r\n    \"userInteraction\": []\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# SRCDS Base Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt -y update\r\napt -y --no-install-recommends install curl lib32gcc1 ca-certificates\r\n\r\ncd \/tmp\r\ncurl -sSL -o steamcmd.tar.gz http:\/\/media.steampowered.com\/installer\/steamcmd_linux.tar.gz\r\n\r\nmkdir -p \/mnt\/server\/steamcmd\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\n# SteamCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server\r\n.\/steamcmd.sh +login anonymous +force_install_dir \/mnt\/server +app_update ${SRCDS_APPID} +quit\r\n\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so",
```
Comment (parkervcp) [inline wreckfest/egg-wreckfest.json]: Install needs to have a user that owns the game to install.

**Pull request closed without merging**